xlml/apis/test_config.py

```diff
@@ -400,8 +402,7 @@ def _from_json_helper(
         setup=setup,
         exports=exports,
         test_command=test_command,
-        # `timeout` is in seconds
-        time_out_in_min=test['timeout'] // 60,
+        time_out=datetime.timedelta(minutes=test['timeout']),
```

Comment:
this is seconds? From JSonnet. Please check change below as well.

Reply:
You are right, thanks for catching that! Updated.